controllers/manifest_controller.go

```diff
@@ -76,7 +76,7 @@ func SetupWithManager(
 	if controllerManagedByManager.Complete(ManifestReconciler(mgr, codec, checkInterval)) != nil {
 		return fmt.Errorf("failed to initialize manifest controller by manager: %w", err)
 	}
-	return nil // never end up here :)
```

Comment:
To reviewer:
a misleading (it should be expected to return nil) and unnecessary comments.

Reply:
True, it was an unnecessary comment on line 79.